README.md

```diff
@@ -1,89 +1,96 @@
 
-## Contributors
+## Contributing to the documentation
 
-Our goal is to provide the Magento community with comprehensive and quality technical documentation. We believe that to accomplish that goal we need experts from the community to share their knowledge with us and each other. We are thankful to all of our contributors for improving Magento documentation.
+Our goal is to provide the Magento community with comprehensive and quality technical documentation. We believe that to accomplish our goal we need experts from the community to share their knowledge with us and each other. We are thankful to all of our contributors for helping to improve the Magento documentation.
 
 ![DevDocs contributors](https://raw.githubusercontent.com/wiki/magento/magento2/images/dev_docs_contributors.png)
 
 ## Building this site
 
-The website uses [GitHub Pages](https://pages.github.com/) tools for building with the [Jekyll](https://jekyllrb.com/) static website generator on top of it.
-The toolkit requires a Ruby environment to be run.
+The website uses [GitHub Pages](https://pages.github.com/) tools for building with the [Jekyll](https://jekyllrb.com/) static website generator, which requires a Ruby environment.
 
-There are several options available depending on what type and level of virtualization you want to use:
+After verifying your [Ruby environment](#ruby-environment), continue with one of the following:
 
-- [Installing the project dependencies locally](#build-using-jekyll)
-- [Using a Docker container](#build-using-docker) (Mac, Linux)
-- [Using a Vagrant virtual machine](#build-using-vagrant)
+-  [Build DevDocs using Jekyll](#local-build-using-jekyll)
+-  [Minimize local build time](#minimize-local-build-time)
+-  [Build DevDocs using a Vagrant virtual machine](#build-using-vagrant)
+-  [Build DevDocs in Windows](#build-devdocs-in-windows) (Not supported)
 
-## Installing the project dependencies locally
+If you have questions, open an issue and ask us. We look forward to hearing from you!
 
-### Prerequisites
+-  [Follow @MagentoDevDocs](https://twitter.com/MagentoDevDocs)
+-  [E-mail us](mailto:DL-Magento-Doc-Feedback@magento.com)
+-  [Visit our documentation site](https://devdocs.magento.com).
 
-#### Ruby
+### Ruby environment
 
-Recommended version: 2.5.
+Jekyll requires a Ruby environment. The Mac and Ubuntu operating systems provide Ruby, but you can choose to upgrade the version or use a version manager to control version scope.
 
-Supported versions: 2.3, 2.4, 2.5.
+-  [Ruby](https://www.ruby-lang.org/en/documentation/installation)—Supported versions: 2.3, 2.4, 2.5
+-  [RubyGems](https://rubygems.org/pages/download)—packaging system for Ruby
+-  [rbenv](https://github.com/rbenv/rbenv)—Optional, but useful for managing one or more Ruby versions concurrently
```

Comment:
Just because they are PHP developers does not mean they don't use different versions of Ruby for other purposes. Most developers I have known are curious folk that like to experiment with multiple technologies. It is not out of the realm of possibility that one would entertain multiple version of anything. Even in our own building we have an underground lair of Python developers...using...virtual environments.